neuraxle/steps/numpy.py

```diff
@@ -51,6 +51,19 @@ def transform_one(self, data_inputs):
     def _concat(self, data_inputs):
         return np.concatenate(data_inputs, axis=-1)
 
+class NumpyConcatenateOnTimeStep(NonFittableMixin, BaseStep):
+    def __init__(self):
+        BaseStep.__init__(self)
+        NonFittableMixin.__init__(self)
+
+    def transform(self, data_inputs):
+        return self._concat(data_inputs)
+
+    def transform_one(self, data_inputs):
+        return self._concat(data_inputs)
+
+    def _concat(self, data_inputs):
+        return np.concatenate(data_inputs, axis=1)
```

Comment:
Could the axis become a constructor argument? So we could have a base class which needs such an argument at construction, and this class would inherit of it and just change the constructor to pass `1` by default. The other step `NumpyConcatenateOuterBatch` just below could also inherit form the same base class, this time with a default arg of `0`.

Reply:
Yeah, I thought about constructing a NumpyConcatenateOnCustomAxis instead of the NumpyConcatenateOnTimeStep. Maybe, it will be more general and more clear than a specific NumpyConcatenateOnTimeStep. Yeah, it could also be the base class. I can make that change if you want to.